packages/pwa-kit-dev/src/utils/script-utils.ts

```diff
@@ -84,6 +85,33 @@ export const getProjectPkg = async (): Promise<Pkg> => {
     }
 }
 
+/**
+ * Get the set of file paths within a specific directory
+ * @param dir Directory to walk
+ * @returns Set of file paths within the directory
+ */
+export const walkDir = async (
+    dir: string,
+    baseDir: string,
+    fileSet?: Set<string>
+): Promise<Set<string>> => {
+    fileSet = fileSet || new Set<string>()
+    const entries: Dirent[] = await readdir(dir, {withFileTypes: true})
+
+    await Promise.all(
+        entries.map(async (entry) => {
+            const entryPath = path.join(dir, entry.name)
+            if (entry.isDirectory()) {
+                await walkDir(entryPath, baseDir, fileSet)
+            } else {
+                fileSet?.add(entryPath.replace(baseDir + path.sep, ''))
```

Comment:
```suggestion
                fileSet?.add(path.relative(baseDir, entryPath))
```
Totally inconsequential, but here's another method that takes care of resolving paths for you!